temporal/api/cloud/namespace/v1/message.proto

```diff
@@ -87,13 +81,20 @@ message Limits {
     int32 actions_per_second_limit = 1;
 }
 
-message AWSPrivateLink {
+message AWSPrivateLinkInfo {
     // The list of principals that are allowed to access the private links on the namespace.
     repeated string allowed_principal_arns = 1;
     // The list of private links, if any, that are associated with the namespace.
     repeated string vpc_endpoint_service_names = 2;
 }
 
+message PrivateConnectivity {
```

Comment:
+1. "Private Connectivity" is a good generic term! It's called PrivateLink in AWS, Private Link in Azure, and Private Service Connect (PSC) in GCP.